data/json/items/gunmod/underbarrel.json

```diff
@@ -514,7 +514,7 @@
     "id": "m26_mass",
     "type": "GUNMOD",
     "name": { "str": "M26-MASS shotgun" },
-    "description": "Modular Accessory Shotgun System, an underbarrel bolt-action shotgun, inspired by \"Masterkey\", and fixed most of its problems.  It can be mounted under the barrel of different rifles, or used as standalone weapon by attachment to a pistol grip or collapsible buttstock module.  It feeds from three or five shot detachable magazines.",
+    "description": "The Modular Accessory Shotgun System, an underbarrel bolt-action shotgun that was inspired by the \"Masterkey\", and fixed most of its predecessor's problems.  It can be mounted under the barrel of different rifles, or used as standalone weapon by attaching it to a pistol grip or collapsible buttstock module.  It feeds from three or five shot detachable magazines.",
```

Comment:
```suggestion
    "description": "The Modular Accessory Shotgun System is an underbarrel bolt-action shotgun that was inspired by the \"Masterkey\", and fixed most of its predecessor's problems.  It can be mounted under the barrel of different rifles, or used as standalone weapon by attaching a pistol grip or collapsible buttstock module.  It feeds from three or five shot detachable magazines",
```

Reply:
Took your comments into account when going with my own edits to this one. It definitely needed another look though, so thank you for bringing it to my attention.